src/app/domain/components/questions/create/create.component.ts

```diff
@@ -61,17 +58,17 @@ export class CreateQuestionsComponent implements OnInit {
     }
 
     submitCallback(values) {
-        values.subject && this.form.controls.subject.setValue(values.subject.trim());
-        values.question && this.form.controls.question.setValue(values.question.trim());
-        values.rounds = this.roundNumber;
+        // TODO FIX THIS ASSIGN
```

Comment:
should this be fixed before we merged (don't  think SSCS will be happy if they cannot create questions)

Reply:
Agreed, I commented for temp, but I had to refactor it for lint to pass.